<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evans</class>
 <widget class="QMainWindow" name="Evans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evans</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #483d8b, stop: 1 #2c255c);
</string>
    </property>
    <widget class="QWidget" name="header" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Vladimir Script</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-weight: bold;
background-color: rgba(0, 0, 0, 0); </string>
      </property>
      <property name="text">
       <string>Evans</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
	border: none;
	border-radius: 15px;
	padding: 8px 16px;
    background-color: rgba(0, 0, 0, 0); 
    border: none;
}

QToolButton:hover {
    background-color: #E0BBFF; /* 薄い紫 */
}</string>
      </property>
      <property name="text">
       <string>⚙</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>25</y>
        <width>531</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>UD Digi Kyokasho N</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>主題曲を選択</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chooseMenu" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>801</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>390</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	border-radius: 15px;
	padding: 8px 16px;
    background-color: rgba(0, 0, 0, 0); 
    border: none;
	color: white;
}

QPushButton:hover {
    background-color: #E0BBFF; /* 薄い紫 */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resource/icon/play.png</normaloff>../resource/icon/play.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="song_Image">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>531</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="music_title">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-weight: bold</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="text">
       <string>使用する楽曲</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="next">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>101</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-weight: bold
</string>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>101</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-weight: bold</string>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
